webpages/settings/style.css

```diff
@@ -133,6 +133,12 @@ h1 {
   vertical-align: middle;
   filter: invert(0) brightness(10);
   margin-inline-end: 8px;
+  user-drag: none;
```

Comment:
![image](https://user-images.githubusercontent.com/61329810/124191097-e5c17380-da88-11eb-8b76-d6b83d80f8ae.png) (caniuse)

so remove all instances of `user-drag` and use `draggable="false"` instead

Reply:
i can remove it but adding `draggable="false"` is uneeded

Reply:
it's only supported in chromium with `-webkit-` and no other browsers support it

why is draggable=false unneeded? that's literally what this does

Reply:
Because -webkit-user-drag works

Reply:
only in chromium

and it's not reccomended